utils/index.d.ts

```diff
@@ -1487,6 +1488,19 @@ export declare class ReadOnlyContent {
  */
 export declare function openReadOnlyContent(node: { label: string, fullId: string }, content: string, fileExtension: string, options?: TextDocumentShowOptions): Promise<ReadOnlyContent>;
 
+/**
+ * Stash a read-only editor so it can be opened by its uri later.
+ * @param node Typically (but not strictly) an `AzExtTreeItem`. `label` is used for the file name displayed in VS Code and `fullId` is used to uniquely identify this file
+ * @param content The content to display
+ * @param fileExtension The file extension
+ */
+export declare function stashReadOnlyContent(node: { label: string, fullId: string }, content: string, fileExtension: string): Promise<ReadOnlyContent>;
+
+/**
+ * Disposes all the read-only contents stashed in memory.
+ */
+export declare function disposeReadOnlyContents(): Promise<void>;
```

Comment:
optional id parameter to dispose specific content?

Reply:
I would like that to be added with the auto-disposal logic.